modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1185,6 +1186,113 @@ definition {
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.setName(formViewName = "My Form View");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+		FormViewBuilder.closeSidebar();
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Select from List");
+
+		FormViewBuilder.save();
+
+		AppBuilderAdmin.switchTab(tabName = "Table Views");
+
+		AppBuilderAdmin.add();
+
+		TableViewBuilder.setName(tableViewName = "My Table View");
+
+		TableViewBuilder.addColumnByDoubleClick(fieldType = "Text");
+
+		TableViewBuilder.addColumnByDoubleClick(fieldType = "Select from List");
+
+		TableViewBuilder.save();
+
+		FormViewBuilder.switchToFormViewTab();
+
+		AppBuilderAdmin.openItem(itemName = "My Form View");
+
+		FormViewBuilder.fieldEllipsisDelete(fieldLabel = "Text");
+
+		FormViewBuilder.validateDeleteUsedObjectMessage();
+
+		FormViewBuilder.validateUsedFormView(formViewName = "My Form View");
+
+		FormViewBuilder.validateUsedTableView(tableViewName = "My Table View");
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Text");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Text");
+
+		FormViewBuilder.deleteFromObjectViaX(fieldName = "Select from List");
+
+		FormViewBuilder.validateDeleteUsedObjectMessage();
+
+		FormViewBuilder.validateUsedFormView(formViewName = "My Form View");
+
+		FormViewBuilder.validateUsedTableView(tableViewName = "My Table View");
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Select from List");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Select from List");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Numeric");
+
+		FormViewBuilder.fieldEllipsisDelete(fieldLabel = "Numeric");
+
+		FormViewBuilder.validateDeleteObjectMessage();
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Numeric");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Numeric");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Multiple Selection");
+
+		FormViewBuilder.deleteFromObjectViaX(fieldName = "Multiple Selection");
+
+		FormViewBuilder.validateDeleteObjectMessage();
+
+		FormViewBuilder.cancelButtonInDeleteFromObject();
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Multiple Selection");
+		FormViewBuilder.validateObjectFieldName(fieldName = "Multiple Selection");
+
+		FormViewBuilder.deleteFromObjectViaX(fieldName = "Multiple Selection");
+
+		FormViewBuilder.validateDeleteObjectMessage();
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Multiple Selection");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Multiple Selection");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Single Selection");
+
+		FormViewBuilder.save();
+
```

Comment:
Hi @tainguyen100 , I would suggest you to validate if the fields are no longer available for others Form Views (Form View 2, for example).